src/routers/healthCheck.js

```diff
@@ -11,4 +12,12 @@ router.get('/_healthcheck', (req, res) => {
   res.status(200).send('EUREKA!');
 });
 
+router.post('/login', authController.login);
+
+router.get('/isAuthenticated', authController.isLoggedIn, (req, res) => {
```

Comment:
why we need this?

Reply:
This route just test a authenticated route.